pkg/diag/colors/windows.go

```diff
@@ -16,6 +16,20 @@
 
 package colors
 
+import "golang.org/x/sys/windows/registry"
+
+func hasVTSupport() bool {
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Windows NT\CurrentVersion`, registry.QUERY_VALUE)
+	if err != nil {
+		return false
+	}
+	maj, _, err := k.GetIntegerValue("CurrentMajorVersionNumber")
+	if err != nil {
+		return false
+	}
+	return maj >= 10
+}
+
 func init() {
-	disableColorization = true
+	disableColorization = !hasVTSupport()
```

Comment:
so no colorization pre-win10?

Reply:
might want to mention that in the CL if that's the case.

Reply:
Correct. IIRC pre-Win10 the terminal did not understand the VT100 control codes we use for colorization.